Pal/src/host/Linux-SGX/sgx_main.c

```diff
@@ -467,7 +467,7 @@ int initialize_enclave (struct pal_enclave * enclave)
 add_pages:
         TRY(add_pages_to_enclave,
             &enclave_secs, (void *) areas[i].addr, data, areas[i].size,
-            areas[i].type, areas[i].prot, areas[i].skip_eextend,
```

Comment:
I would prefer keeping this `skip_eextend` option instead of making the assumption that only file mappings are measured (what if we want to add a measured private mapping?).

Reply:
Fine with me.